### PR TITLE
Update perl-Bootloader test for sle micro

### PR DESCRIPTION
**Pull request by cedvid**
- Related ticket: https://progress.opensuse.org/issues/153871
- Verification run: 
  - SLE https://openqa.suse.de/tests/overview?build=os-autoinst%2Fos-autoinst-distri-opensuse%2319023
  - openSUSE https://openqa.opensuse.org/tests/overview?build=os-autoinst%2Fos-autoinst-distri-opensuse%2319023

### CODE REVIEW

**Review by ricardobranco777 — APPROVED** [review]

File: tests/console/perl_bootloader.pm [hunk 1]
```diff
@@ -39,11 +41,20 @@ sub run {
...
         }
     }
-    assert_script_run 'pbl --install';
-    assert_script_run 'pbl --config';
-    power_action('reboot', textmode => 1);
-    $self->wait_boot;
-    select_serial_terminal;
+
+    if (is_transactional) {
+        trup_call 'run pbl --install';
```
Comment (mloviska) [inline tests/console/perl_bootloader.pm:46]: I think you can use `-c #no` in order to chain tr-up commands and avoid reboots

Comment (dzedro) [reply tests/console/perl_bootloader.pm:46]: Hacks :smile: would be nice to have `trup_call` support chain, can you please open issue ?

Comment (mloviska) [reply tests/console/perl_bootloader.pm:46]: But I bet we should have more VRs than 2 especially TW

**Review by foursixnine — APPROVED** [review]

**Review by dzedro — APPROVED** [review]

**Comment by DimStar77** [comment]
causes a failure here:

https://openqa.opensuse.org/tests/4080567#step/perl_bootloader/107

that test does intentionally not use grub2 - it's sdboot based. As such, pbl fails to configure grub

**Comment by cedvid** [comment]
> causes a failure here:
> 
> https://openqa.opensuse.org/tests/4080567#step/perl_bootloader/107
> 
> that test does intentionally not use grub2 - it's sdboot based. As such, pbl fails to configure grub

Noted, thank you. I will not load the module for sdboot based tests: [PR](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19096)

**Pull request merged**